### PR TITLE
feat: extend fields

### PR DESCRIPTION
**Pull request by lingsamuel**
### Description

Add more fields.

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible

### CODE REVIEW

File: pkg/api/apisix/types/types.go [hunk 1]
```diff
@@ -33,27 +33,34 @@ type Vars [][]StringOrSlice
...
+	Upstream        *Upstream         `json:"upstream,omitempty" yaml:"upstream,omitempty"`
+	UpstreamID      string           `json:"upstream_id,omitempty" yaml:"upstream_id,omitempty"`
 	ServiceID       string           `json:"service_id,omitempty" yaml:"service_id,omitempty"`
 	Plugins         Plugins          `json:"plugins,omitempty" yaml:"plugins,omitempty"`
-	PluginConfigId  string           `json:"plugin_config_id,omitempty" yaml:"plugin_config_id,omitempty"`
+	PluginConfigID  string           `json:"plugin_config_id,omitempty" yaml:"plugin_config_id,omitempty"`
 	FilterFunc      string           `json:"filter_func,omitempty" yaml:"filter_func,omitempty"`
+	Script          string           `json:"script,omitempty" yaml:"script,omitempty"`
+	ScriptID        string           `json:"script_id,omitempty" yaml:"script_id,omitempty"`
+	Status          int              `json:"status,omitempty" yaml:"status,omitempty"`
```
Comment (tao12345666333) [inline pkg/api/apisix/types/types.go:60]: It's also for API7, right?

File: test/cli/suites-basic/dump.go [hunk 2]
```diff
@@ -25,12 +25,14 @@ routes:
   - PUT
   name: route1
   service_id: svc1
+  status: 1
```
Comment (tao12345666333) [inline test/cli/suites-basic/dump.go:28]: Should we remove this field when dump configuration?

File: pkg/api/apisix/types/types.go (hunk 1 above)
Comment (lingsamuel) [reply pkg/api/apisix/types/types.go:60]: script and script_id are fields created by dashboard for plugin orchestration

File: test/cli/suites-basic/dump.go (hunk 2 above)
Comment (lingsamuel) [reply test/cli/suites-basic/dump.go:28]: I am not sure. Maybe users want to use dump to generate/update their configuration file? Keep these fields may avoid the diff of default values.

**Review by tao12345666333 — APPROVED** [review]

**Pull request merged**